packages/client/modules/teamDashboard/components/UnpaidTeamModal/UnpaidTeamModal.tsx

```diff
@@ -54,6 +72,52 @@ const UnpaidTeamModal = (props: Props) => {
   const {team} = viewer
   if (!team) return null
   const {name: teamName, organization, lockMessageHTML} = team
+
+  const {name: organizationName} = organization
+
+  const {id: orgId, billingLeaders, name: orgName} = organization
+  const [firstBillingLeader] = billingLeaders
+  const billingLeaderName = firstBillingLeader?.preferredName ?? 'Unknown'
+  const email = firstBillingLeader?.email ?? 'Unknown'
+  const isALeader = billingLeaders.findIndex((leader) => leader.id === viewerId) !== -1
+
+  const goToBilling = (upgradeCTALocation: UpgradeCTALocationEnum) => {
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation
+    })
+    history.push(`/me/organizations/${orgId}`)
+  }
+
+  if (organization.lockedAt) {
```

Comment:
-1 Can we add an `Upgrade CTA Viewed` event here as well with `upgradeCTALocation` set?

Reply:
@tghanken added, can you check?